server/utils/TestingUtils/MongoDbManager.js

```diff
@@ -0,0 +1,57 @@
+const { MongoClient } = require('mongodb');
+const { MongoMemoryServer } = require('mongodb-memory-server');
+
+// Extend the default timeout so MongoDB binaries can download
+jest.setTimeout(60000);
+
+// List all of your collection names here - I'll add some examples
+const COLLECTIONS = ['completeCollection'];
+
+class DBManager {
+  constructor() {
+    this.db = null;
+    this.server = new MongoMemoryServer(
+        {
+            instance : {
+                dbName : 'rpaFrameworkCommands'
+            }
+        });
+    this.connection = null;
+  }
+
+  // Spin up a new in-memory mongo instance
+  async start() {
+    const url = await this.server.getUri();
+    process.env.MONGODB_URI = url;
+
+    this.connection = await MongoClient.connect(url, {
+      useNewUrlParser: true,
+      useUnifiedTopology: true,
+    });
+
+    this.db = this.connection.db(await this.server.getDbName());
+    await this.db.createCollection('completeCollection');
+    let testDocuments = [
+        {"Application":"MS Excel","Task":"Open Application","Code":"Open Application","inputVars":{"visible":"Boolean","display_alerts":"Boolean"}},
```

Comment:
ah I guess these are dummies to populate the DB? So my comment above about the magic numbers might be obsolete. Am I right?

Reply:
yes you are